lib/collect-jsons.js

```diff
@@ -15,40 +16,41 @@ module.exports = function collectJSONS(options) {
         throw new Error(`There were issues reading JSON-files from '${options.jsonDir}'.`);
     }
 
-    if (files.length === 0) {
-        throw new Error(`No JSON files found in '${options.jsonDir}'. NO REPORT CAN BE CREATED!`);
-    }
-
-    files.map(file => {
-        // Cucumber json can be  empty, it's likely being created by another process (#47)
-        const data = fs.readFileSync(file).toString() || "[]";
-        JSON.parse(data).map(json => {
-            if (options.metadata && !json.metadata) {
-                json.metadata = options.metadata;
-            } else {
-                json = Object.assign({
-                    "metadata": {
-                        "browser": {
-                            "name": "not known",
-                            "version": "not known"
-                        },
-                        "device": "not known",
-                        "platform": {
-                            "name": "not known",
-                            "version": "not known"
+    if (files.length > 0) {
+        files.map(file => {
+            // Cucumber json can be  empty, it's likely being created by another process (#47)
+            const data = fs.readFileSync(file).toString() || "[]";
+            JSON.parse(data).map(json => {
+                if (options.metadata && !json.metadata) {
+                    json.metadata = options.metadata;
+                } else {
+                    json = Object.assign({
+                        "metadata": {
+                            "browser": {
+                                "name": "not known",
+                                "version": "not known"
+                            },
+                            "device": "not known",
+                            "platform": {
+                                "name": "not known",
+                                "version": "not known"
+                            }
                         }
-                    }
-                }, json);
-            }
-            jsonOutput.push(json)
+                    }, json);
+                }
+                jsonOutput.push(json)
+            });
         });
-    });
-
-    if (options.saveCollectedJSON) {
-        const file = path.resolve(options.reportPath, 'merged-output.json');
-        fs.ensureDirSync(options.reportPath);
-        jsonFile.writeFileSync(file, jsonOutput, {spaces: 2});
+    
+        if (options.saveCollectedJSON) {
+            const file = path.resolve(options.reportPath, 'merged-output.json');
+            fs.ensureDirSync(options.reportPath);
+            jsonFile.writeFileSync(file, jsonOutput, {spaces: 2});
+        }
+    
+        return jsonOutput;
     }
 
-    return jsonOutput;
+    console.log(chalk.yellow(`No JSON files found in '${options.jsonDir}'. NO REPORT CAN BE CREATED!`));
```

Comment:
Looks great!

Can you add `WARNING:` in front of it so the message will be 

`WARNING: No JSON files found in '${options.jsonDir}'. NO REPORT CAN BE CREATED!`